src/cloudant/database.py

```diff
@@ -489,6 +489,32 @@ def unshare_database(self, username):
         resp.raise_for_status()
         return resp.json()
 
+    def index(self):
+        """
+        GET /db/_index Returns existing indexes
+        """
+        pass
+
+    def find(self, selector, **kwargs):
+        """
+        POST /db/_find
```

Comment:
Is it really a POST on the other end? That's sad ... a search should really be a GET.